src/lib/converter/plugins/IncludePlugin.ts

```diff
@@ -108,8 +150,264 @@ export class IncludePlugin extends ConverterComponent {
             }
         }
     }
+
+    getRegion(
+        refl: Reflection,
+        file: string,
+        ext: string,
+        textPart: string,
+        text: string,
+        target: string,
+    ) {
+        const regionTagsList = regionTagREsByExt[ext];
+        let found: string | false = false;
+        for (const [startTag, endTag] of regionTagsList) {
+            const start = text.match(startTag(target));
```

Comment:
We should regex-escape `target` here, there's an `escapeRegExp` function in `src/lib/utils/general.ts`

Reply:
Good point. Done in my latest commit.